tests/cli/test_cli_serve.py

```diff
@@ -24,15 +26,32 @@ def test_cli_serve_help():
     assert result.exit_code == 0
 
 
-@pytest.mark.skip
-def test_cli_serve_up():
+# @pytest.mark.skip
+def test_cli_serve():
 
     from nos.cli.cli import app_cli
-    from nos.tests.utils import NOS_TEST_DATA_DIR
+    from nos.test.utils import NOS_TEST_DATA_DIR, NOS_TEST_IMAGE
 
+    config_path = NOS_TEST_DATA_DIR / "hub/custom_model/config.yaml"
     # Move to the test data directory
-    path = NOS_TEST_DATA_DIR / "hub/custom_model/config.yaml"
+    with path_ctx(config_path.parent):
+        serve_up_result = runner.invoke(app_cli, ["serve", "up", "-c", "config.yaml", "-d"])
+        assert serve_up_result.exit_code == 0
 
-    with path_ctx(path.parent):
-        result = runner.invoke(app_cli, ["serve", "-c", "config.yaml", "up"])
-        assert result.exit_code == 0
+    from nos.client import Client
```

Comment:
maybe move the rest of this into a separate test?

Reply:
I removed the upload file part here since it was for capturing the upload path error in the first place, but the error was due to something else which doesn't have to be in here. But the WaitForServer seems nessesary here, since serve down right after serve up might causing error I think. @outtanames do you think this is good enough?

Reply:
SGTM.